### PR TITLE
add rss feed testing-stage

### PR DESCRIPTION
**Pull request by mehdi-alouane**

### CODE REVIEW

File: backend/app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@posts = Post.all(:select => "title, author, id, content, posted_at", :order => "posted_at DESC", :limit => 20)
```
Comment (mkhatib) [inline backend/app/controllers/posts_controller.rb]: What are posts? We don't have a `Post` model. Our article model is called `Article`.

Also this doesn't seem like a valid controller code. Where's the controller class. See `articles_controller.rb` for an example and there is probably where you want to add the `respond_to` block to respond to both json and rss.

File: backend/app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= auto_discovery_link_tag(:rss, "http://manshar.com") %>
```
Comment (mkhatib) [inline backend/app/views/layouts/application.html.erb]: Can you elaborate what is this for? Not sure we entirely need this.

**Comment by mkhatib** [comment]
@mehdi-alouane if you meant to keep this for testing stage, you can keep it in a branch in your own fork and then once you think it is ready for production you can send a pull request again to `manshar:master`. For now I'll close this as we keep `master` production ready (i.e. master is always deployed to production)

**Comment by mkhatib** [comment]
Btw, thanks a lot for sending a pull request! Happy to chat about this more and provide guidance on where to go from here to get this in. Or we can discuss this either here or on our mailing list `core@manshar.com`

File: backend/app/views/layouts/application.html.erb (hunk 2 above)
Comment (mehdi-alouane) [inline backend/app/views/layouts/application.html.erb]: to get browsers to auto-detect your rss

**Pull request closed without merging**